README.md

```diff
@@ -125,6 +125,31 @@ Topic information: Topic: /chatter | Type: std_msgs/String | Count: 9 | Serializ
                    Topic: /my_chatter | Type: std_msgs/String | Count: 18 | Serialization Format: cdr
 ```
 
+### Overriding QoS Profiles
+
+When starting a recording or playback workflow, you can pass a YAML file that contains QoS profile settings for a specific topic.
+The YAML schema for the profile overrides is a dictionary of topic names with key/value pairs for each QoS policy:
+
+```yaml
+topic_name: str
+  qos_policy_name: str
```

Comment:
There are a limited number of policies (6), perhaps you could list out the ones that are accepted here? It'll probably be more helpful than this schema.